webapp/pages/profile/_id/_slug.vue

```diff
@@ -251,7 +251,7 @@
           </template>
         </masonry-grid>
         <div
-          v-if="hasMore"
+          v-if="hasMore && posts.length >= pageSize"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-07T08:49:47Z" title="Monday, October 7th 2019, 10:49:47 am +02:00">Oct 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -251,7 +251,7 @@
           </template>
         </masonry-grid>
         <div
-          v-if="hasMore"
+          v-if="hasMore && posts.length >= pageSize"
```


Is this the bug? Was the `hasMore` not enough? I can see that this condition will be `false` initially. But how relates that to the bug?